CMakeLists.txt

```diff
@@ -386,8 +376,8 @@ IF(WIN32)
 ENDIF()
 
 ADD_CUSTOM_TARGET(DEPENDENCIES_FOR_PACKAGE)
-IF("${WITH_SSL}" STREQUAL "")
-  MESSAGE(STATUS "Setting deafault value for WITH_SSL for libmariadb build to ON")
+IF(NOT WITH_SSL)
```

Comment:
Doesn't this make impossible to build without encryption support?  i.e. even if someone explicitly configures with -DWITH_SSL=Off, it will be still reverted to build with openssl. My intention was to use default only if no value is provided by a user